sdk/python/touca/cli/server.py

```diff
@@ -157,14 +156,20 @@ def parser(cls, parser: ArgumentParser):
             dest="dev",
             help="Install for development environment",
         )
+        parser.add_argument(
+            "--test",
+            action="store_true",
+            dest="test",
+            help="Install for test environment",
+        )
```

Comment:
Adding a `--test` flag to `touca server install` so that we can use it in the GitHub Actions CI workflow `self-hosted-install.yml` which uses `docker-compose.test.yml`.